project/src/page/utils/getPlaceCards.js

```diff
@@ -0,0 +1,16 @@
+import PlaceCard from '../../components/place-card/place-card';
+import React from 'react';
+
+export function getPlaceCards (image, info, pageType) {
```

Comment:
Во время работы с реактом, привычнее видеть такого рода функции как отдельные компоненты.

```suggestion
export function PlaceCards (image, info, pageType) { return <Fragment>..... image.map....
```

С одной стороны это дело вкуса. Но, с другой стороны, рано или поздно возникают всякие новые потребности. Нужно отслеживать кто из детей hover, или selected. и прочиев вопросы состояния списка, как целого. А это проще делать через useStae и подобные ему крючки

Reply:
Я про это помню! И именно так и сделаю! Просто пока не вижу еще общей картины приложения.